src/map/packets/message_system.h

```diff
@@ -26,6 +26,53 @@
 
 #include "basic.h"
 
+// Found in ROM/27/76.dat
+// clang-format off
+enum MSGSYSTEM_ID : uint16
```

Comment:
`enum class` for type safety

Reply:
our other one was also not class enum. I shall attempt to adjust both of them and amend this PR.

Reply:
wound up just doing the one I added, because converting MSGBASIC wound up being too large to get in one go.